test/system/roles/public_health/dashboard/dashboard.rb

```diff
@@ -67,6 +61,41 @@ def start_export(workflow, export_type, action)
     end
   end
 
+  def export_excel_single_monitoree(patient_label)
+    search_for_and_view_patient('all', patient_label)
+    click_on 'Download Excel Export'
+    @@public_health_export_verifier.verify_excel_single_monitoree(patient_label.split('_')[1].to_i)
+  end
+
+  def export_custom(settings)
+    click_on 'Export'
+    click_on settings[:preset] || 'Custom Format...'
+
+    # Choose which records to export
+    choose "select-monitoree-records-#{settings[:records]}" if settings[:records].present?
+
+    # Choose which elements to export
+    settings[:elements]&.each_value do |data_type|
+      data_type[:checked]&.each do |label|
+        # find("rct-#{data_type}-elements").click_on(label)
```

Comment:
Why are these commented out?

Reply:
ah, those shouldn't be there. had to try out a couple of different things to get capybara to click on the checkboxes, will remove

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/1f13143f8968feff3c7e9ada6139b47e5cd4ecff#diff-79c5de538df633a3e0b34e6a90f124a3787c350fb0c0b6ff9b2dac2060907960L80